addons/website_sale/models/payment_token.py

```diff
@@ -0,0 +1,21 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class PaymentToken(models.Model):
+    _inherit = 'payment.token'
+
+    def _get_available_tokens(self, *args, is_express_checkout=False, **kwargs):
+        """ Override of `payment` to return the tokens in case of express checkout.
```

Comment:
This is the opposite of what it does: it does *not* return the tokens when the order is being paid with express checkout.